integ-test/src/test/resources/correctness/expressions/arithmetics.txt

```diff
@@ -1,3 +1,7 @@
+6 * 2
+6 *2
+6* 2
+6*2
```

Comment:
can you add example tests for all operators without spaces (they should work too)
`+ - / %`

Reply:
Added Test Cases